gerrymandering/gerrymandering.ipynb

```diff
@@ -116,17 +110,29 @@
    "outputs": [],
    "source": [
     "x_range, y_range = ((-13884029.0, -7453303.5), (2818291.5, 6335972.0)) # Continental USA\n",
-    "shade_defaults = dict(x_range=x_range, y_range=y_range, x_sampling=10, y_sampling=10, width=1200, height=682,\n",
-    "                      color_key=color_key, aggregator=ds.count_cat('race'),)\n",
-    "shaded = datashade(census_points, **shade_defaults)\n",
+    "\n",
+    "shaded = df.hvplot.points(\n",
+    "    'easting', 'northing',\n",
+    "    datashade=True,\n",
+    "    aggregator=ds.count_cat('race'),\n",
+    "    cmap=color_key,\n",
+    "    xlim=x_range,\n",
+    "    ylim=y_range,\n",
+    "    dynspread=True,\n",
+    "    height=800,\n",
+    "    width=1000,\n",
+    "    data_aspect=1,\n",
+    "    tiles='EsriImagery'\n",
```

Comment:
I think `geo=True` might be needed if I'm not mistaken

Reply:
IIRC it was working without it, not sure how 🙃

Reply:
Okay, geo=True isn't needed here because the data is already in easting northing (GOOGLE_MERCATOR) coordinates.
```
    "shaded = df.hvplot.points(\n",
    "    'easting', 'northing',\n",
```

However, to be explicit, and what I usually do, is you can define
`crs=ccrs.GOOGLE_MERCATOR` and `projection=ccrs.GOOGLE_MERCATOR`

Reply:
I don't think you need to specify `projection`, since tiles are always in that projection?